### PR TITLE
BiG EVAL (Certified Connector)

### PR DESCRIPTION
**Pull request by boltth**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/BiG EVAL/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,513 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "BiG EVAL",
+    "description": "BiG EVAL Data Quality Automation platform for data validation and data test automation.",
```
Comment (v-EgorKozhadei) [inline certified-connectors/BiG EVAL/apiDefinition.swagger.json:5]: @boltth description should contain information about the main value and purpose offered by the connector. Please make it more concise. You can refer to the documentation for more [information](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector).

File: certified-connectors/BiG EVAL/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,513 @@
...
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://bigeval.com/company/privacy/"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Data;Business Intelligence"
+    }
+  ],
+  "host": "bigeval-lttbo01.eu.ngrok.io",
```
Comment (v-EgorKozhadei) [inline certified-connectors/BiG EVAL/apiDefinition.swagger.json:27]: @boltth can you please confirm that this is a production environment host URL?

File: certified-connectors/BiG EVAL/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,513 @@
...
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "name": {
+                  "type": "string",
+                  "description": "name",
+                  "title": "",
```
Comment (v-EgorKozhadei) [inline certified-connectors/BiG EVAL/apiDefinition.swagger.json:75]: @boltth please fill in empty titles.

**Review by v-EgorKozhadei — COMMENTED** [review]
Hello @boltth,

Please check comments after the review.

Let us know if you have any questions.

File: certified-connectors/BiG EVAL/apiDefinition.swagger.json (hunk 2 above)
Comment (boltth) [reply certified-connectors/BiG EVAL/apiDefinition.swagger.json:27]: BiG EVAL is an on-premise solution and every customer will configure his own public URL. Therefore no public available production-URL exists. We will change this to anything like "bigeval.mycompany.com"

File: certified-connectors/BiG EVAL/Readme.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# BiG EVAL Connector
```
[the remark this answers is not in the document]
Comment (boltth) [reply certified-connectors/BiG EVAL/Readme.md]: done

File: certified-connectors/BiG EVAL/Readme.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+# BiG EVAL Connector
+BiG EVAL provides a powerful way of integrating data testautomation and/or data quality management into business process automation. 
+This connector makes you able to automatically trigger Microsoft Power Automate flows when specific events happen in BiG EVAL's data validation processes.
+
```
[the remark this answers is not in the document]
Comment (boltth) [reply certified-connectors/BiG EVAL/Readme.md:4]: done

File: certified-connectors/BiG EVAL/apiDefinition.swagger.json (hunk 1 above)
Comment (boltth) [reply certified-connectors/BiG EVAL/apiDefinition.swagger.json]: done

File: certified-connectors/BiG EVAL/apiDefinition.swagger.json (hunk 3 above)
Comment (boltth) [reply certified-connectors/BiG EVAL/apiDefinition.swagger.json]: done

**Comment by boltth** [comment]
@v-EgorKozhadei Requested Changes were done. Please re-review the PR.

**Review by v-EgorKozhadei — APPROVED** [review]

**Comment by v-EgorKozhadei** [comment]
Hello @boltth, 

I hope you are doing well. 

Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-updates) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file. 

After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6. 

Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector.  

We expect all tests to be completed within 1-2 weeks. 

If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions. 

Please let me know if you have any questions or run into any problems while creating or updating your submission. 

Thank you very much for working with us.

**Pull request merged**